ofctrl/fgraphFlow.go

```diff
@@ -2153,13 +2225,9 @@ func (self *Flow) generateFlowMessage(commandType int) (flowMod *openflow13.Flow
 			flowMod.AddInstruction(writeInstruction)
 		}
 		if self.gotoTable != nil {
-			gotoTableInstruction := openflow13.NewInstrGotoTable(*self.gotoTable)
+			gotoTableInstruction := openflow15.NewInstrGotoTable(*self.gotoTable)
 			flowMod.AddInstruction(gotoTableInstruction)
 		}
-		if self.meter != nil {
```

Comment:
reminder

Reply:
Actually, I remember you mentioned that meter is working as an action in OF1.5 instead of instruction. But I didn't see code to set meter action in a FlowMod message. If I miss something, could you help point out? 

Anyway, meter is set in the caller (antrea) when it is expected to be used in a flow. But I didn't find the consumption.